report_template/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html data-bs-theme="light" lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+    <title>AllSpice Component Report</title>
+    <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
+    <link rel="stylesheet" href="assets/fonts/fontawesome-all.min.css">
+    <link rel="stylesheet" href="assets/css/Navbar-Right-Links-Dark-icons.css">
+</head>
+
+<body style="border-color: var(--bs-gray-400);">
+    <div class="row" style="background: var(--bs-secondary-text-emphasis);padding: 10px;margin-bottom: 20px;">
+        <div class="col"><span style="font-size: 1.2em;font-weight: bold;color: var(--bs-body-bg);">&nbsp;<img class="rounded-circle" src="assets/img/leaf_green_bg.png" style="height: 36px;">&nbsp; Component Report - DigiKey API</span></div>
+    </div>
+	{% for part in bom %}
+    <div style="margin-bottom: 20px;">
+        <div style="width: 48px;margin-left: 30px;color: var(--bs-emphasis-color);background: var(--bs-gray-500);text-align: center;height: 26px;"><span style="font-size: 12px;background: var(--bs-gray-500);height: 0px;">{{ loop.index }}</span></div>
+		{% if 'Obsolete' == part.lifecycle_status  %}
+        <div style="margin-left: 30px;margin-right: 30px;border-style: solid;border-color: var(--bs-red);border-radius: 3px;padding-bottom: 0px;">
+        {% elif 'Not For New Designs' == part.lifecycle_status %}
+        <div style="margin-left: 30px;margin-right: 30px;border-style: solid;border-color: var(--bs-red);border-radius: 3px;padding-bottom: 0px;">
+		{% else %}
+        <div style="margin-left: 30px;margin-right: 30px;border-style: solid;border-color: var(--bs-gray-500);border-radius: 3px;padding-bottom: 0px;">
+		{% endif %}
+            <div class="row" style="font-size: 12px;color: rgb(255,255,255);margin-right: 0px;margin-left: 0px;">
+                <div class="col" style="background: var(--bs-secondary-text-emphasis);padding-top: 5px;padding-bottom: 5px;"><span>{{ part.associated_refdes }}</span></div>
+            </div>
+			{% if part.mfr_name is not none %}
+            <div class="row g-0">
+                <div class="col-auto text-center" style="padding-right: 20px;padding-left: 20px;border-style: none;border-color: var(--bs-gray-400);"><img src="{{ part.photo_url }}" style="height: 88px;margin-left: 10px;"><a href="{{ part.datasheet_url }}" target="_blank">
+                        <p style="text-align: center;font-size: 10px;padding-bottom: 5px;"><i class="far fa-file-pdf" style="font-size: 20px;color: var(--bs-form-invalid-color);"></i></p>
```

Comment:
We're only using font awesome for this one icon AFAICT, is it worth adding all of its assets for this?

Reply:
I'm able to replace the font awesome full library with just the SVG I need for that icon - will make the change.